[base_bc] crash when creating a community channel

### Description

When creating a community channel and leave description empty, submitting this request (creating the channel) crashes the app

```
/Users/anastasiya/status-desktop/src/nim_status_client.nim(146) nim_status_client
/Users/anastasiya/status-desktop/src/nim_status_client.nim(143) mainProc
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qguiapplication.nim(29) exec
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(49) qobjectCallback
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(34) onSlotCalled
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/core/macros.nim(568) onSlotCalled
/Users/anastasiya/status-desktop/src/app_service/service/message/service.nim(431) onMarkAllMessagesRead
/Users/anastasiya/status-desktop/src/app/core/eventemitter.nim(46) emit
/Users/anastasiya/status-desktop/src/app/modules/main/chat_section/controller.nim(67) :anonymous
/Users/anastasiya/status-desktop/src/app/modules/main/chat_section/private_interfaces/module_controller_delegate_interface.nim(20) onMarkAllMessagesRead
/Users/anastasiya/status-desktop/src/app/modules/main/chat_section/module.nim(374) onMarkAllMessagesRead
/Users/anastasiya/status-desktop/src/app/modules/main/chat_section/module.nim(353) updateNotifications
/Users/anastasiya/status-desktop/vendor/nimbus-build-system/vendor/Nim/lib/pure/collections/tables.nim(262) []
Error: unhandled exception: key not found: 0x0286c8af5357ce1285a206de564fb5e25c5838d0c9db318f04c66157822b98aac791e4f239-b21c-4242-92b5-7d5da9ff1b5d [KeyError]
make: *** [run-macos] Error 1
```

Expected: community channel description is mandatory , so there should be a validation first


when adding a community channel with description the app also crashes:

```
INF 2022-01-13 14:24:44.977+03:00 shutting down...                           topics="status-app" tid=1037772 file=nim_status_client.nim:121
Traceback (most recent call last)
/Users/anastasiya/status-desktop/src/nim_status_client.nim(146) nim_status_client
/Users/anastasiya/status-desktop/src/nim_status_client.nim(131) mainProc
/Users/anastasiya/status-desktop/src/app/boot/app_controller.nim(245) delete
/Users/anastasiya/status-desktop/src/app_service/service/message/service.nim(76) delete
/Users/anastasiya/status-desktop/vendor/nimqml/src/nimqml/private/qobject.nim(64) delete
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Error 1
anastasiya@MacBook-Pro-Anastasiya
```

--- a/src/app/modules/main/chat_section/module.nim
+++ b/src/app/modules/main/chat_section/module.nim
@@ -350,7 +350,8 @@ proc updateNotifications(self: Module, chatId: string, unviewedMessagesCount: in
   # update model of this module (appropriate chat from the chats list (chats model))
   self.view.chatsModel().updateNotificationsForItemOrSubItemById(chatId, hasUnreadMessages, unviewedMentionsCount)
   # update child module
-  self.chatContentModules[chatId].onNotificationsUpdated(hasUnreadMessages, unviewedMentionsCount)
+  if (self.chatContentModules.contains(chatId)):
+    self.chatContentModules[chatId].onNotificationsUpdated(hasUnreadMessages, unviewedMentionsCount)
   # update parent module
   let (sectionHasUnreadMessages, sectionNotificationCount) = self.view.chatsModel().getAllNotifications()
   self.delegate.onNotificationsUpdated(self.controller.getMySectionId(), sectionHasUnreadMessages, sectionNotificationCount)

